DataFormats/PatCandidates/interface/Muon.h

```diff
@@ -418,6 +422,7 @@ namespace pat {
     /// Muon MVA
     float mvaValue_;
     float lowptMvaValue_;
+    float mvaIDValue_;
     float softMvaValue_;
```

Comment:
it could be that the differences in https://github.com/cms-sw/cmssw/pull/36179#issuecomment-1009751651 are from the order of the new value.

Note that the FWLite-based plots are made using the old/reference first. So, the dictionary for the new version is not working.
In the past for cases with confusions like this, I'd get have a local build and run the comparison script in the reverse order of inputs.

Reply:
That makes sense, thanks for your help!

We have compared the values of the MVAs before/after the PR in several samples and all are exactly the same, so everything seems to be in order. Let us know if there's anything else we can do.